src/shared/components/case-editor/case-edit-page/case-edit-page.component.ts

```diff
@@ -91,6 +91,7 @@ export class CaseEditPageComponent implements OnInit, AfterViewChecked {
 
   applyValuesChanged(valuesChanged: boolean): void {
     this.formValuesChanged = valuesChanged;
+    this.currentPageIsNotValid();
```

Comment:
Not sure why this is being called here as it doesn't capture the returned value or do anything with it...?